server/src/main/java/io/druid/server/http/DatasourcesResource.java

```diff
@@ -200,6 +201,53 @@ public Response deleteDataSource(
     return Response.ok().build();
   }
 
+  /*
```

Comment:
`@Deprecated` should be added to the above method (in the java code) also [here](https://github.com/druid-io/druid/pull/2276/files#diff-9ad19c35d703fbc035ab9725dcbb9fafR173)